test/stripe/customer_test.rb

```diff
@@ -46,7 +46,7 @@ class CustomerTest < Test::Unit::TestCase
     context "#create_subscription" do
       should "create a new subscription" do
         customer = Stripe::Customer.retrieve("cus_123")
-        subscription = customer.create_subscription(plan: "silver")
+        subscription = customer.create_subscription(items: [{ plan: "silver" }])
```

Comment:
I am a bit surprised by this change. `plan` is allowed on customer creation/update. the `/v1/customers/cus_XXXX/subscription` is an old endpoint and I don't think it should expect `items` as a parameter and expects `plan` instead.

Reply:
Yeah, I think at some point we updated its implementation to use the new endpoint:

```
    def create_subscription(params = {}, opts = {})
      resp, opts = request(:post, subscriptions_url, params, opts)
      initialize_from({ subscription: resp.data }, opts, true)
      subscription
    end


    def subscriptions_url
      resource_url + "/subscriptions"
    end
```

Reply:
Hm, yes ... good point. It's `/v1/customers/cus_123/subscriptions` specifically in this test.

Possibly it's a change that happened in the backend at some point?

Reply:
SG. Thanks.